lib/declarations.d.ts

```diff
@@ -199,10 +199,21 @@ interface IProjectData extends IDictionary<any> {
 interface IProjectPropertiesService {
 	getProjectProperties(projectFile: string, isJsonProjectFile: boolean, frameworkProject: Project.IFrameworkProject): IFuture<IProjectData>;
 	completeProjectProperties(properties: any, frameworkProject: Project.IFrameworkProject): boolean;
-	updateProjectProperty(projectData: any, mode: string, property: string, newValue: any): IFuture<void>;
+	updateProjectProperty(projectData: any, configurationSpecificData: IProjectData, mode: string, property: string, newValue: any): IFuture<void>;
 	normalizePropertyName(property: string, projectData: IProjectData): string;
 	getValidValuesForProperty(propData: any): IFuture<string[]>;
 	getPropertiesForAllSupportedProjects(): IFuture<string>;
+	/**
+	 * Removes property from the project and validates the result data. In case @configurationSpecificData is present, removes the property only from it,
```

Comment:
@configurationSpecificData is not part of the method's params anymore. Remove the reference to it.